src/components/Layout/Menu.js

```diff
@@ -27,6 +27,7 @@ export function Menu() {
           <Link to="/transcripts/">Transcripts</Link>
           <Link to="http://jobs.reactiflux.com/">Jobs</Link>
           <Link to="/learning/">Learning</Link>
+          <Link to="/contact/">Contact</Link>
```

Comment:
I think I'm gonna strike this, think we should publicize it at the same time as some kind of announcement post.
```suggestion
```